scripts/main/build.js

```diff
@@ -101,9 +103,14 @@ build.album = function (data, disabled = false) {
 
 	let html = lychee.html`
 			<div class='album ${disabled ? `disabled` : ``} ${data.is_nsfw && lychee.nsfw_blur ? `blurred` : ``}'
+				id='album-${data.id}'
 				data-id='${data.id}'
 				data-nsfw='${data.is_nsfw ? `1` : `0`}'
-				data-tabindex='${tabindex.get_next_tab_index()}'>
+				data-tabindex='${tabindex.get_next_tab_index()}'
+				draggable='${album.isSmartID(data.id) ? "false" : "true"}'
+				ondragstart='startAlbumDrag(event)'
+				ondragover='event.preventDefault()'
+				ondrop='dropAlbum(event)'>
```

Comment:
Smart albums can be neither the source nor the destination for such dragging, so I think at least `ondrop` should be conditional as well? Also, it's not just smart albums but also the tag albums.

Also, it would be nice to provide a visual feedback when hovering over a destination album while dragging (I think the same visual effect that we have for regular hover would work fine).